src/js2xbox/events.cpp

```diff
@@ -23,10 +23,15 @@
 #include <unistd.h>
 #include <stdexcept>
 #include <linux/input.h>
+#include <glob.h>
 
 #include "../logger.h"
 #include "../globals.h"
 
+static const char *EVDEV_WILDCARD =
+	"/dev/input/by-path/platform-*-joypad-event-joystick";
+static const char *EVDEV_GENERIC =
+	"/dev/input/by-path/platform-joypad-event-joystick";
```

Comment:
This actually exists on a device I own (a R36s clone), and will never match the glob, so I figured we should have it as a fallback.

Reply:
Will just `/dev/input/by-path/platform-*-event-joystick` work everywhere?